float32 velocity   # KPH
float32 steering   # Clockwise (Left : -, Right : +)
int32 brake        # ON, OFF